### ISSUE TITLE
Replace Jade Admin UI Templates with EJS

### ISSUE THREAD

**Issue by JedWatson** [keystonejs/keystone#3155]
Note: I've already done the work, realised I should have opened an issue for visibility, so I'm opening this for the sake of everyone being across it. Sorry! Next time I'll do it in the right order 😅 

Using Jade for the Admin UI is a hangover from the pre-React days, and it's really overkill for the simple templates we're serving now. EJS is much lighter and faster, so I wanted to move them to that.

Also, the Admin UI templates were previous rendered by `keystone.render` in `lib/core` which seriously needed a review / cleanup, and is the only part of the Admin UI not now encapsulated in the `/admin` folder.

I've rewritten the templates in EJS, removed the old Jade ones, implemented _just_ the locals we need in a much simpler way in the `signin` and `index` routes themselves, and removed the old `lib/core/render.js` module.

Should be a lot easier to follow what's going on now for new developers, the Admin UI is properly encapsulated at last, and we have less dependencies!   🎉

**Comment by mxstbr** [comment]
Noo, this breaks what I was doing in #3150. God dammit Jed!

**Comment by JedWatson** [comment]
... and that is why I opened an issue! Go me. Feeling totally sheepish, sorry @mxstbr 

On the up side, doesn't this simplify things a bit? there's not that many conflicts...

**Comment by mxstbr** [comment]
> On the up side, doesn't this simplify things a bit?

Yeah yeah, fishing for compliments. :grin:

It's alright, but I could've waited before starting it.

**Comment by Pylipala** [comment]
Yes, I also prefer EJS template. 

However if I use `yo keystone` to generate a template site. There is no ejs option but only `Jade, Swig, Nunjucks or Handlebars`

How should I work around this?

**Comment by inyoka** [comment]
> Yes, I also prefer EJS template.
> 
> However if I use `yo keystone` to generate a template site. There is no ejs option but only `Jade, Swig, Nunjucks or Handlebars`
> 
> How should I work around this?

Did you ever find a workaround for this?  Strange EJS is not an option still.
